src/core_plugins/kibana/index.js

```diff
@@ -84,7 +90,26 @@ module.exports = function (kibana) {
       ingest(server);
       search(server);
       settings(server);
+
+      registerCoreTranslations();
     }
   });
 
 };
+
+function registerCoreTranslations()
+{
+  const rootDir = kibanaPackage.__dirname;
+
+  //Add translation dirs for the core plugins here
+  const corePluginTranslationDirs = [rootDir + '/src/ui/i18n'];
```

Comment:
You can use the fromRoot util to accomplish this https://github.com/Bargs/kibana/blob/2422a988e5f23010a5f45ee04a7ffb451cf01d72/src/utils/from_root.js